examples/experimental/dagster-airlift/dagster_airlift/in_airflow/mark_as_migrating.py

```diff
@@ -0,0 +1,34 @@
+import json
+import sys
+
+from airflow import DAG
+
+from ..migration_state import AirflowMigrationState
+
+
+def mark_as_dagster_migrating(
+    migration_state: AirflowMigrationState,
+) -> None:
+    """Alters all airflow dags in the current context to be marked as migrating to dagster.
+    Uses a migration dictionary to determine the status of the migration for each task within each dag.
+    Should only ever be the last line in a dag file.
+    """
+    # get global context from above frame
+    global_vars = sys._getframe(1).f_globals  # noqa: SLF001
```

Comment:
The grumpy software engineer wants to make the user pass in globals (similar to the way that `file_relative_path`) to that the user knows what is going on and so that we can test it.

```python
def mark_as_dagster_migrating(
    globals_dict: Dict,
    migration_state: AirflowMigrationState,
) -> None:

mark_as_dagster_migrating(globals(), AirflowMigrationState())
```

People copy and paste anyway and this provides some more flexibiliy for users. You can imagine users who have done their own post processing pass in a similar way and this would allow them to compose this with that.

Reply:
Convinced.